test/suite/test_join01.py

```diff
@@ -207,10 +303,15 @@ def test_join(self):
         for cfga in [ '', bloomcfg1000, bloomcfg10000 ]:
             for cfgb in [ '', bloomcfg1000, bloomcfg10000 ]:
                 for do_proj in [ False, True ]:
-                    #self.tty('cfga=' + cfga +
-                    #         ', cfgb=' + cfgb +
-                    #         ', doproj=' + str(do_proj))
-                    self.join_common(cfga, cfgb, do_proj, False)
+                    for do_nested in [ False, True ]:
+                        for order in range(0, 4):
+                            #self.tty('cfga=' + cfga +
+                            #         ', cfgb=' + cfgb +
+                            #         ', doproj=' + str(do_proj) +
+                            #         ', donested=' + str(do_nested) +
+                            #         ', order=' + str(order))
+                            self.join_common(cfga, cfgb, do_proj, do_nested,
+                                             False, order)
```

Comment:
@ddanderson, this is now 5 levels deep: would it make sense to replace with scenarios?